commands/project/download.ts

```diff
@@ -127,8 +131,9 @@ exports.builder = yargs => {
       describe: i18n(`${i18nKey}.options.dest.describe`),
       type: 'string',
     },
-    buildNumber: {
-      describe: i18n(`${i18nKey}.options.buildNumber.describe`),
+    build: {
+      describe: i18n(`${i18nKey}.options.build.describe`),
+      alias: ['build-id'],
```

Comment:
Should we also leave `build-number` as an alias to make this a less breaking change?

Reply:
All these audit changes are already going to be breaking so I'd personally prefer to just rip the bandaid here since we don't refer to `build number` anywhere else in projects